### ISSUE TITLE
Follow up on issue #811

### ISSUE THREAD

**Issue by raghavrmadya** [filecoin-project/notary-governance#826]
## Issue Description
https://github.com/filecoin-project/notary-governance/issues/811

The notaries in question were requested to provide evidence of how their signing is aligned with their due diligence plan. Some did so in the comments but given the number of comments on this popular thread, things got lost.

## Impact
Pause of signing for notaries in question

## Proposed Solution(s)
Post evidence already shared n this thread for the community to review. 

### Timeline
Feb 15th 2023 - Feb 20th 2023 - Community discussion
Feb 21st 2023 - Consensus announced at T&T WG call

### Technical dependencies 
N/A

## Risks and mitigations 
Slowdown of LDN TTD

**Comment by raghavrmadya** [comment]
If the community has suggestions on how the notaries in question can earn back trust if evidence fails community consensus, please post in the thread as well.

**Comment by cryptowhizzard** [comment]
I am not satisfied with the evidence provided and i am not satisfied with the change in behaviour.

[This issue](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1130) for example has two notaries who ( on the last datacap approval ) approved a clearly abusive LDN application. This goes for Metawave and Unionlabs (IPFSCN) and this just happened days ago. It should stop inmediately.

[This issue](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1362) is still open. The signature is still not revoked where it should be revoked. This goes for Metawave.

Above notary's applied to be a R4 notary and stated in their own notary application that they would provide KYC and duediligence. Non of this is being done and until they will strongly advise to have them paused.

![afbeelding2](https://user-images.githubusercontent.com/36404435/219061360-d7bc5cca-66bc-44fc-90eb-adb6c6960697.png)

**Comment by herrehesse** [comment]
I am not agreeing with both the provided evidence and the apparent lack of substantial change in behaviour.

I must express my deep frustration and disappointment with the notaries in question. In my opinion, they have completely eroded the trust of their respective communities, and as a result, they should not be allowed to sign any further applications. At an absolute minimum, they must demonstrate good faith by taking measures to rectify their abusive behaviour that has been documented. It is irrelevant whether their actions were intentional or not, as the outcome remains the same. Notaries have a responsibility to exercise due diligence and possess the necessary skillset to conduct tests on applications.

This is a list of the notaries who should be immediately barred from signing any further applications until they have regained the trust of their respective communities.

| Notary Name | Signed CID Abusive Applications |
| --- | --- |
| Tom-OriginStorage | 30 |
| swatchliu | 26 |
| liyunzhi-666 | 23 |
| newwebgroup | 21 |
| MetaWaveInfo | 18 |
| MRJAVAZHAO | 18 |


I suggest this separate list below for those notaries who require close monitoring to ensure they are adequately performing their due diligence, but can still sign applications. Most of these notaries acted in good faith, but did not know they where singing an application with high CID abuse. 

| Notary Name | Signed CID Abusive Applications |
| --- | --- |
| kernelogic | 15 |
| psh0691 | 15 |
| PluskitOfficial | 13 |
| stcouldlisa | 13 |
| UnionLabs2020 | 11 |
| BlockMakeronline | 11 |
| Defil2022 | 10 |

I would like to propose two additional points. Firstly, I recommend that notaries who remain idle and have no utility to the community be discontinued as a notary after a period of at least three months of absence, or a shorter duration if deemed appropriate. Secondly, I suggest maintaining a list of notaries who have been signing applications without adequate due diligence. These notaries should only be re-given the community's trust after they have performed due diligence on 50 new applications.

**Comment by kernelogic** [comment]
@herrehesse are the number of CID abusive LDNs counting from all time or only after the CID checker bot comes out? There was no way to check CID sharing before the bot comes out so signatures done before that shouldn't count as intentional abuse.

@raghavrmadya I have signed hundreds of LDNs of all time, actually I am the #1 notary signed the most LDNs according to PL's statistics. If we look at the percentage of good vs bad, I should actually get an award for my hard work right?

**Comment by xixiott** [comment]
https://github.com/filecoin-project/notary-governance/issues/825#issuecomment-1432471905

**Comment by kernelogic** [comment]
https://github.com/filecoin-project/notary-governance/issues/825#issuecomment-1432479269

**Comment by herrehesse** [comment]
@kernelogic Agreeing with you here, the numbers are factual so I can not change them, but most (close to all) of your signatures were before the CID checker report. This does not change the fact that close to none due diligence have been done on those signed applications. That is something we hope to permanently improve in 2023.

**Comment by liyunzhi-666** [comment]
In this list, all signings occurred before the CID checker came out. So this shouldn't be considered a signature abuse.I agree with @kernelogic .

**Comment by Chris00618** [comment]
It 's too crazy!

![image](https://user-images.githubusercontent.com/105741726/219399650-5a740659-e126-46b4-945d-1be1374563e2.png)

**Comment by raghavrmadya** [comment]
I would appreciate if this thread was focused on notaries that were in question for 811. I do not see any evidence shared yet by the 6 notaries in question. If community members would like to flag other notaries, kindly open a new issue so that we can stay focused.

**Comment by stcouldlisa** [comment]
@raghavrmadya Since the end of #811, I have actively participated in notary meetings where galen expressed the opinion that each notary has his own investigation protocol and it is enough to follow his own protocol. Therefore, I signed a total of 4 LDNs after 811 closed, and I retrieve all of them before signing them, and give the retrieveable proofs on GitHub at.

#1266-1268 (fildrivelabs): This is the first time I signed for fildrivelabs, this round of signing was done by me and cryptowhizzard, reason for signing: retrievable, reasonable sp assignment.
#925 (F3): If I remember correctly, the client demonstrated it at the notary meeting. Also, I made sure the sp was retrievable before signing. I believe cryptowhizzard supports F3, also because I saw their demo at the notary meeting.
#1508 (Kernelogic - Sentinel3): public dataset, retrievable, reasonable SP assignment.
#1471: This was the first signature for a client, done by cryptowhizzard and I. The client sent the domain email, disclosed the number of SPs and addresses, and filled out the cryptowhizzard form.

**Comment by Carohere** [comment]
I'd like to request further DD on [ #1078](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1078). @Tom-OriginStorage should be put on hold until everything is clear.

**Comment by newwebgroup** [comment]
Hey @raghavrmadya 
From #811 so far, we have not signed any LDN.
During this time, we have been trying to learn more about the Fil+ rules, actively participating in notary meetings and T&T group meetings, and learning more by talking to other notaries (e.g., how do you verify that an SP can support a search via Lotus? etc.)
We will continue to refine our KYB investigation methods to help us be able to get back to normal faster.

**Comment by ipfscn** [comment]
hello,
We never signed any LDNs after #811,
We will also not be signing any LDNs for the next 3 months（May 18, 2023）
Thanks

**Comment by Tom-OriginStorage** [comment]
hi  @raghavrmadya 
signing LDNs, and asked for an explanation for the 15 recently signed LDNs. In #811, we gave detailed explanations for the 15 recently signed LDNs
Among them, 5 signatures are the first round of LDN, all of which require the applicant to complete KYB according to the rules at that time or other notaries have completed KYB, and the other 10, 9 of which robot inspections are in compliance with the rules, of which #638 has 9 Sector CID sharing, communication is an operational error, which can basically be ignored
For detailed information, you can go to #811 to view,
In #811, if our interpretation can be recognized by the community, we can continue to sign, just like @raghavrmadya  said, there was no notary objecting at that time, so after #811 ended, we made the signature rules In-depth study, and some communication with other notaries,
Two weeks later, on February 13th, we returned to the community and started due diligence and signing of the LDN. I signed a total of 4 LDNs, 3 of which were due diligence by the rest of the notary, and I conducted due diligence again. ,
They all comply with the rules and have no CID sharing, and they can all be retrieved; the other one has been reviewed by myself, and its packaging basically complies with the rules. There are 5.7T of CID sharing in the same project, and the retrieval is also normal;

**Comment by Tom-OriginStorage** [comment]
> I'd like to request further DD on [ #1078](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1078). @Tom-OriginStorage should be put on hold until everything is clear.

no problem thank you for your attention

**Comment by NDLABS-OFFICE** [comment]
Hi @raghavrmadya , thanks for raising a proposal that allows the community to have a complete discussion and to bring closure to proposal #811.
First of all, welcome everyone to review the form of how ND signed for the last 15 LDNs. I have also marked on the form why we are willing to support and sign. Please see attached.
Also, I would like to explain the two LDNs I signed after the #811 here.
1. #1008: First of all, the client completed the KYB email, the client's information is complete and the data sample content is detailed. and the client selected more than 4 SPs and the checker report is good. what's more, I have checked all SPs of the LDN to see whether it is reachable and retrievable. I signed after the result was good.
2. #1508: First of all, this project is a public dataset submitted by the kernelogic team, who are active in the community and in a high level of trust.  The checker report looks well with decentralized SPs and the number of nodes is pretty much. Even though there is CD sharing, it is in a reasonable way.  Meanwhile, since this LDN has a large number of SPs, I randomly checked some SPs' retrievability, and all showed well. so I signed.
（Regarding why ND has done the signatures in the past few days, I need to explain why. We attended every session of the governance call and the answer we got at the call was that the community was allowing the notary to do signatures if a reasonable review was done. So after  #811 was not going through, we took a period of hiatus before we started signing again.
During this time, Dcent, the active notary in the community, DMed me about whether ND needed to be able to sign after 50 LDN reviews, and I explained to Hid why we could sign. As it turned out, as RG stated in here, we were allowed to sign during that period, provided we conducted due diligence on the LDNs, which we did not misunderstand.）
The above is ND's explanation as an active and challenged notary. In addition, ND would like to take this opportunity to express some of our own views on the community situation.
First, the standards of review for notaries are not mandatory and may differ from notary to notary, and they are always changing with the different tools available. (which is great and is a sign of keep moving of Fil+ community)
At present, we have communicated with other notaries and ND's own review standards are as follows：
1、The client conducts KYC email confirmation to ensure that the application is submitted by the client's own will.
2, the information of the application is complete and has a reasonable data sample, we also consider the data backup situation of the LDN to make a comprehensive judgment.
3, the client has a reasonable explanation for the community members' questions, and the community members have no new questions.
4、After the check bot is launched, we mainly refer to the check bot inspection, such as the client sharing too many CIDs or storage is not in compliance with the rules, notaries will require an explanation in a public way, and will not sign without a reasonable explanation.
5、We will test the connectivity of SPs for recent LDNs, in addition, by communicating with Xinan Xu, we believe that it is best if one LDN has more than 4 different operators.
6. The notary attempt to retrieve the CID of the data. The precondition is that SPs can support retrieval, and then the notary will perform a sampling comparison of the stored data with the sample data.
The standard of review is also based on the current standard of review of most notaries in the community. In addition, we are also calling for a discussion of Notary Proposal #790 on slack, in order to have a basic consensus of review among notaries and community members.
We also noticed that the Dcent team has a VPN-like inspection tool, which is worth promoting, but since the tool is not open source and most notaries don't have such tools, it is difficult for us to inspect the VPN of clients. In addition, regarding the sp policy in China is that many small and medium-sized sp is not willing to disclose their contact information and other information, this is something I hope the governance team can take into consideration.
I would like to take this opportunity to apologize to many of the clients who have chatted with me privately in Slack, as we are currently on a signature moratorium, so we can only review but not support signatures to move their storage business forward.

Finally, I would like to say that Filecoin is a friendly community, we have members from different countries, and we all have different cultures but we all share a common goal, which is to make the Filecoin project go further and better. 
Just like the news of FVM's launch this year, it has given everyone long-awaited confidence. However, the current situation in the community does have some "voices". We hope that the community can help notaries and the community to be more transparent and reasonable by building a consensus on notary review standards as well as the launch of open source tools like check bot and VPN check to complete the due diligence reasonably. Cause the only constant is continuous change.
[ND.Signature.Record.xlsx](https://github.com/filecoin-project/notary-governance/files/10774855/ND.Signature.Record.xlsx)

**Comment by cryptowhizzard** [comment]
> > I'd like to request further DD on [ #1078](https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1078). @Tom-OriginStorage should be put on hold until everything is clear.
> 
> no problem thank you for your attention ，I gave the answer below 1078

That answer was however not answering the question itself. I asked if Xhash is / was your customer. A clear yes / no is missing. Secondly you indicated that you spoke to Xhash themselves and that they indicated that their storage plan was not in line with the FIL+ rules for the first allocation(s) . This communication is / was not forwarded to the community for review, to conclude there is no proof that due diligence is or was done.

"They all comply with the rules and have no CID sharing, and they can all be retrieved; the other one has been reviewed by myself, and its packaging basically complies with the rules. There are 5.7T of CID sharing in the same project, and the retrieval is also normal;"

This is simply not true. See above. Bending the rules does not make an applicant comply with the rules. A famous saying -> A little bit poison in your drink is still poison.

**Comment by Tom-OriginStorage** [comment]
1: I can tell you that XHASH is not a customer of OeiginStorage, you can find out what they do. I have explained your question politely, please don't question without reason, don't think that only your rules are correct, if you have time, you can answer the Decent/Speedium shared CID on slack;
https://filecoinproject.slack.com/archives/C0405HANNBT/p1676654179383529

![image](https://user-images.githubusercontent.com/103015469/220147445-4673ab8f-c63b-4274-a60d-eaea885be728.png)


2: Regarding the current rules of FIL+, I believe everyone is very clear that the final packaging of LDN needs to meet the standard. The first packaging is only 50T. Not everyone can auction the quota like dcent; use the quota to earn income;

**Comment by cryptowhizzard** [comment]
> @cryptowhizzard
> What I mean is that they only have a 50T quota at the moment, and it is a bit premature for us to draw conclusions on the distribution of data. They promised that the new SP will be reflected in the second round of 100T quota.
> If you want to see the later results, please help them sign and let him continue to show you the later results

Well, i asked for the data onboarding plan where the next data will be stored. It still is not present because you did not do proper due diligence. I asked if this is a customer of you ( it would explain the kyc already done and agreement on storage ) .. but since you confirm this is not the case i must presume this signature came out of the blue? Why do you sign without DD on someone you don’t know and against your own commitments in your application?

If the data onboarding plan would show the 100% selfdealing *before* or on the moment you signed then this would be more credible. Now it isn’t.

**Comment by Tom-OriginStorage** [comment]
He is not my client, he contacted me on WeChat through some people, because we are all in the blockchain industry, I communicated this issue with them in detail on WeChat, I am completely sure of the authenticity of their company Yes, they are also well-known in the blockchain industry. As for the problem you mentioned that there are currently only 3 SPs, they will solve it in the second round, because the current 50T quota in the first round has been used up. The latest rule for FIL+ is to store 4 copies, but isn’t this just the beginning? The entire LDN just stores 1% of the data. Can you see what I mean?

**Comment by Tom-OriginStorage** [comment]
1: The first round has been assisted by a notary to complete the KYC,
2: WeChat records in China can be used as legal evidence;

**Comment by Patapon0702** [comment]
> hi @raghavrmadya signing LDNs, and asked for an explanation for the 15 recently signed LDNs. In #811, we gave detailed explanations for the 15 recently signed LDNs Among them, 5 signatures are the first round of LDN, all of which require the applicant to complete KYB according to the rules at that time or other notaries have completed KYB, and the other 10, 9 of which robot inspections are in compliance with the rules, of which #638 has 9 Sector CID sharing, communication is an operational error, which can basically be ignored For detailed information, you can go to #811 to view, In #811, if our interpretation can be recognized by the community, we can continue to sign, just like @raghavrmadya said, there was no notary objecting at that time, so after #811 ended, we made the signature rules In-depth study, and some communication with other notaries, Two weeks later, on February 13th, we returned to the community and started due diligence and signing of the LDN. I signed a total of 4 LDNs, 3 of which were due diligence by the rest of the notary, and I conducted due diligence again. , They all comply with the rules and have no CID sharing, and they can all be retrieved; the other one has been reviewed by myself, and its packaging basically complies with the rules. There are 5.7T of CID sharing in the same project, and the retrieval is also normal;

Hey bro, what's wrong with my application form? #638

**Comment by Tom-OriginStorage** [comment]
> > hi @raghavrmadya signing LDNs, and asked for an explanation for the 15 recently signed LDNs. In #811, we gave detailed explanations for the 15 recently signed LDNs Among them, 5 signatures are the first round of LDN, all of which require the applicant to complete KYB according to the rules at that time or other notaries have completed KYB, and the other 10, 9 of which robot inspections are in compliance with the rules, of which #638 has 9 Sector CID sharing, communication is an operational error, which can basically be ignored For detailed information, you can go to #811 to view, In #811, if our interpretation can be recognized by the community, we can continue to sign, just like @raghavrmadya said, there was no notary objecting at that time, so after #811 ended, we made the signature rules In-depth study, and some communication with other notaries, Two weeks later, on February 13th, we returned to the community and started due diligence and signing of the LDN. I signed a total of 4 LDNs, 3 of which were due diligence by the rest of the notary, and I conducted due diligence again. , They all comply with the rules and have no CID sharing, and they can all be retrieved; the other one has been reviewed by myself, and its packaging basically complies with the rules. There are 5.7T of CID sharing in the same project, and the retrieval is also normal;
> 
> Hey bro, what's wrong with my application form? #638

Sorry it's LDN 638, I've changed it

**Comment by Carohere** [comment]
> He is not my client, he contacted me on WeChat through some people, because we are all in the blockchain industry, I communicated this issue with them in detail on WeChat, I am completely sure of the authenticity of their company Yes, they are also well-known in the blockchain industry. As for the problem you mentioned that there are currently only 3 SPs, they will solve it in the second round, because the current 50T quota in the first round has been used up. The latest rule for FIL+ is to store 4 copies, but isn’t this just the beginning? The entire LDN just stores 1% of the data. Can you see what I mean?
> 
> I promise that I will follow up #1078 Data storage reaches at least 4 SPs

@Tom-OriginStorage, can you share the DD process you did for #1078 directly? I think communication evidence would be the best way to clear what @cryptowhizzard flagged above.

**Comment by Neal-fil** [comment]
These notaries continue to claim "unintentional" mistakes despite being proven to be self encapsulated, undue kyc, etc. Community inclusivity does not mean all mistakes should be excused. If such violations will be accepted, think of what R4 with 100+ noatris will become.

**Comment by raghavrmadya** [comment]
Thank you for the discussion on this topic. As outlined above, this issue will be closed today. Based on the discussion above and compliance with the request for evidence, ND Labs, Newwebgroup, STCloud, Tom - Origin storage are no longer under review and can continue to sign LDNs. I understand and truly appreciate the concerns that active community members have flagged here. 

That said, the T&T WG expects that the aforementioned notaries might receive some questions on their signing but I would encourage ND Labs, Newwebgroup, STCloud, Tom - Origin storage to continue to sign and provide supporting evidence rather than pause signing. 

I understand that some community members might still be unsatisfied by the signing evidence provided by the aforementioned notaries and such diversity of opinion is representative of a healthy community imo. 

I will be sharing this on the T&T WG Call.

**Comment by raghavrmadya** [comment]
I do not see any evidence shared by ipfscn and gate.io and as such, they are still paused from signing. Not signing is not a solution or answer to an allegation. If you believe you did the right thing, kindly share your voice with the community.

**Comment by ipfscn** [comment]
In the last 15 IPFSCN signatures, 6 were for large datasets and 9 were for company-type signatures. The following are the details for each signature:
Flying Swallow #983, 1104, 1105: 4 times signed in total, twice for #983. The reason for signing was due to the cooperation with over 10 SPs, the allocation of SPs was reasonable and ensured decentralization, and the backup was healthy.
#1523 and 1524: Both were first-time signatures for large datasets from ND, a listed company. The decision to temporarily support was based on the belief that ND was capable of handling large datasets.
#963: Signed once for the first round. The client provided a domain email and cooperative SPs. The requested amount of 1.5P was reasonable, leading to support for the first time.
#1217: Signed once for the first time. The client had a good qualification with over 200 sales outlets and more than 300 employees serving top 500 companies. The decision to support was based on the verification of the client's identity through a domain email.
#1025: Signed for the first round. The client requested a reasonable amount of 1P and provided a domain email, cooperative SPs and backup numbers, leading to support.
#1085: Signed for the first round after verifying the following information: domain email, cooperative SPs capable of meeting retrieval needs, and at least 4 nodes.
#947: In the second round, 10 SPs were cooperated with, the allocation of SPs was reasonable, and the bot checks showed normal results.
#915: In the second round, the allocation of SPs was reasonable, no CID sharing was present, and the maximum SP ratio did not exceed 21%.
#1097: In the third round, the customer cooperated with 10 SPs, the allocation was reasonable, no CID sharing was present, and the bot check during the signature showed a healthy result.
#638: In the fourth round, RG decentralized the quota and the customer cooperated with 8 SPs. The node allocation ratio was in compliance with FIL+ rules. At that time, the signature bot had not been developed yet.
#888: In the second round, RG decentralized the quota and the second round signature was carried out. The allocation of SPs was reasonable. At that time, the signature bot had not been developed yet.

**Comment by ipfscn** [comment]
![WX20230221-234826@2x](https://user-images.githubusercontent.com/84571713/220393053-47eef29b-14ee-438a-b30a-e936f5e692c3.png)

**Comment by ipfscn** [comment]
@raghavrmadya We recorded a video, but since the content of the file is too large to upload on GitHub, I will send it to you privately.

**Comment by ipfscn** [comment]
Sorry, I still haven't found a proper way to upload videos, so I wrote a text version

**Comment by ipfscn** [comment]
@raghavrmadya  Here is our video link：  https://youtu.be/ttMBp3xnHxg

**Comment by ipfscn** [comment]
Hello everyone, I recorded the video about 1 month ago, I found that GitHub could not upload it, because the video was too big, so I sent it to @raghavrmadya  and @Kevin-FF-USA and @philippbanhardt about a month ago,

**Comment by ipfscn** [comment]
> @raghavrmadya , hello We never signed any LDNs after #811, We also not be signing any LDNs for the next 3 months（May 18, 2023） Thanks

Thanks for everyone's attention, we also always pay attention to the filecoin community.

Finally, our commitment will not change and I will not sign for the next three months.
